app/jobs/send_analytics/daily_stats_job.rb

```diff
@@ -0,0 +1,13 @@
+module SendAnalytics
+  class DailyStatsJob < CronJob
+    self.cron_expression = "0 3 * * *" # every day at 3am
+
+    queue_as :analytics
+
+    def perform(date: Date.yesterday)
+      Rails.logger.info "Sending daily stats CSV for #{date}..."
```

Comment:
@d-a-v-e Are you using `info` so that we can set this level in TEST/DEV (or a level that picks it up)? So it won't be adding to the log in PROD

Reply:
this won't go to prod logs?